spec/schemas/_core.rank_eval.yaml

```diff
@@ -168,6 +168,8 @@ components:
           $ref: '_common.yaml#/components/schemas/IndexName'
         _score:
           type: number
+        _type:
```

Comment:
All these things should be references, like https://github.com/opensearch-project/opensearch-api-specification/blob/3981d332cc830c1547ece4f3eea9ea3399040126/spec/schemas/_core.get.yaml#L12.